scripts/receive_meteor.sh

```diff
@@ -176,59 +176,50 @@ if [ "$METEOR_RECEIVER" == "rtl_fm" ]; then
     fi
   fi
 
-  log "Decoding in progress (QPSK to BMP)" "INFO"
-  ${IMAGE_PROC_DIR}/meteor_decode_qpsk.sh "${qpsk_file}" "${AUDIO_FILE_BASE}" >> $NOAA_LOG 2>&1
-  
-  sleep 10
+  log "Running MeteorDemod do demodulate QPSK file, rectify (spread) images, create heat map and composites and convert them to JPG" "INFO"
+  $METEORDEMOD -t "$TLE_FILE" -f jpg -i "${qpsk_file}" >> $NOAA_LOG 2>&1 #Dodao sam >> $NOAA_LOG 2>&1
 
-  rm "${qpsk_file}"
+  log "Removing QPSK, GCP and BMP files" "INFO"
+  rm "${qpsk_file}" *.gcp *.bmp
 
-  if [ -f "${AUDIO_FILE_BASE}.dec" ]; then
-    if [ "${SUN_ELEV}" -lt "${SUN_MIN_ELEV}" ]; then
-      log "I got a successful ${FILENAME_BASE}.dec file. Decoding APID 68" "INFO"
-      ${IMAGE_PROC_DIR}/meteor_apid68_decode.sh "${AUDIO_FILE_BASE}.dec" "${IMAGE_FILE_BASE}-122" >> $NOAA_LOG 2>&1
-      $CONVERT $FLIP -negate "${IMAGE_FILE_BASE}-122.bmp" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
-    else
-      log "I got a successful ${FILENAME_BASE}.dec file. Creating false color image" "INFO"
-      ${IMAGE_PROC_DIR}/meteor_false_color_decode.sh "${AUDIO_FILE_BASE}.dec" "${IMAGE_FILE_BASE}-122" >> $NOAA_LOG 2>&1
-	  $CONVERT $FLIP "${IMAGE_FILE_BASE}-122.bmp" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
-    fi
+  log "Annotating images and creating thumbnails" "INFO"
+  counter=1
+  for i in *.jpg
+    do
+      ${IMAGE_PROC_DIR}/meteor_normalize_annotate.sh "$i" "$i" 100 >> $NOAA_LOG 2>&1
+      ${IMAGE_PROC_DIR}/thumbnail.sh 300 "$i" "${i%.jpg}-thumb-122-rectified.jpg" >> $NOAA_LOG 2>&1
 
-    log "Rectifying image to adjust aspect ratio" "INFO"
-    python3 "${IMAGE_PROC_DIR}/meteor_rectify.py" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
+      mv "$i" "${IMAGE_FILE_BASE}-${counter}-122-rectified.jpg"
+      mv "${i%.jpg}-thumb-122-rectified.jpg" "${IMAGE_THUMB_BASE}-${counter}-122-rectified.jpg"
 
-    log "Annotating images and creating thumbnails" "INFO"
-    ${IMAGE_PROC_DIR}/meteor_normalize_annotate.sh "${IMAGE_FILE_BASE}-122-rectified.jpg" "${IMAGE_FILE_BASE}-122-rectified.jpg" 100 >> $NOAA_LOG 2>&1
-    ${IMAGE_PROC_DIR}/thumbnail.sh 300 "${IMAGE_FILE_BASE}-122-rectified.jpg" "${IMAGE_THUMB_BASE}-122-rectified.jpg" >> $NOAA_LOG 2>&1
-    rm "${IMAGE_FILE_BASE}-122.bmp"
-    rm "${AUDIO_FILE_BASE}.bmp"
-    rm "${AUDIO_FILE_BASE}.dec"
+      push_file_list="$push_file_list ${IMAGE_FILE_BASE}-${counter}-122-rectified.jpg "
+      ((counter++))
```

Comment:
Why use counter and not `$i`?

Reply:
Since MeteorDemod creates anywhere from 4 to 6 images and counter just creates new names that have the same base, but only differ in that counter number.

Reply:
but doesn't counter always have the same value as `$i` (maybe `$i + 1`)? you are looping with a variable `$i` and then incrementing the counter at the end of each loop.

Reply:
You didn't understand my way of thinking. $i holds image name, $counter is just an ordinal number which is appended to image filenames. That way, I can differentiate images from the same pass. It's the same with NOAA. NOAA appends enchamcements like -MSA, -MCIR etc. while Meteor appends -1, -2 etc.

Reply:
oh I see, then may I suggest you change the variable name from `$i` to something more descriptive like `$imagename` my brain saw `i` and thought integer.